Dockerfile

```diff
@@ -0,0 +1,39 @@
+FROM ubuntu:xenial as zuul
+
+RUN echo "deb http://ppa.launchpad.net/openstack-ci-core/bubblewrap/ubuntu xenial main" >> /etc/apt/sources.list \
+  && apt-get update \
+  && apt-get -y install curl python3.5 python3-setuptools ca-certificates git build-essential zookeeperd libssl-dev libffi-dev libxml2-dev libxslt-dev python3-dev python python-setuptools python-dev \
+  && apt-get -y --allow-unauthenticated install bubblewrap \
+  && easy_install pip \
+  && easy_install3 pip \
+  && pip3 install lxml
+
+# py2-requirements.txt and py3-requirements.txt contains packages required by Tungsten Fabric CI
+RUN cd /tmp \
+  && git clone https://github.com/progmaticlab/zuul.git -b contrail/feature/zuulv3 \
```

Comment:
This Dockerfile is already inside the repo, it shouldn't need to clone anything again. Why not just use the COPY statement?